Source/VaRest/Public/VaRestSubsystem.h

```diff
@@ -15,10 +15,11 @@ DECLARE_DYNAMIC_DELEGATE_OneParam(FVaRestCallDelegate, UVaRestRequestJSON*, Requ
 USTRUCT()
 struct FVaRestCallResponse
 {
-	GENERATED_BODY()
+	GENERATED_USTRUCT_BODY()
```

Comment:
This should be `GENERATED_BODY` as `GENERATED_USTRUCT_BODY` is old and deprecated.